pkgs/by-name/lo/logana/package.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "logana";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "emilycares";
+    repo = pname;
+    tag = version;
+    hash = "sha256-Ke5QTJutGB4f4tnH8CekDXVA1CDU9jkIlGiBhnLxb5c=";
+  };
+
+  cargoHash = "sha256-usn4tk4fdiu5dvbwF/8HnJOWH5OWtETXBHpUj/0GbZw=";
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
https://github.com/NixOS/nixpkgs/issues/208242